app/controllers/blood_pressures_controller.rb

```diff
@@ -10,7 +10,7 @@ def new
   def create
     @blood_pressure = BloodPressure.new(blood_pressure_params)
     @blood_pressure.save 
-    redirect_to blood_pressures_path
+    redirect_to client_path(1)
```

Comment:
client_path(1) ? ---  try client_path(blood_pressures_params[:client_id] to redirect to right client